src/main/java/use_case/assisted_mode/hit/probability/ProbabilityStandCalculator.java

```diff
@@ -23,10 +20,9 @@
 /**
  * The class that calculates the probability of winning a game of BlackJack with a current hand.
  */
-public class ProbabilityStandInteractor implements ProbabilityInteractorInterface, ProbabilityStandInputBoundary {
+public class ProbabilityStandCalculator implements ProbabilityInteractorInterface {
 
     private final GameDataAccessObject gameDataAccessObject;
-    private final ProbabilityStandOutputBoundary standPresenter;
     private final Map<Integer, Integer> unknownCards;
```

Comment:
I would add some documentation to indicate that the values in the maps represent frequencies of cards with the given key as their value